src/configuration/evars.md

```diff
@@ -23,7 +23,25 @@ Changes syntax flavor for disassembler between Intel and AT&T. At the moment, th
 
 ### asm.pseudo
 
-A boolean value to choose a string disassembly engine. "False" indicates a native one, defined by the current architecture, "true" activates a pseudocode strings format; for example, it will show `eax=ebx` instead of a `mov eax, ebx`.
+A boolean value to choose a string disassembly engine. "False" indicates a native one, defined by the current architecture, "true" activates a pseudocode strings format. For example, it'll transform :
```

Comment:
What is a "string disassembly engine". Isn't clear

Reply:
I'm not sure too, it was already there.
How about:
`A boolean value to set the pseudo syntax in the disassembly` ?

Reply:
Much better :)
If you don't understand something, you would want to consider re-phrasing it in better words. Just as you did now <3